### DISCUSSION TITLE
helper script fails to DL

### DISCUSSION THREAD

**Discussion by firepower9966** [Guilouz/Creality-Helper-Script-Wiki#465] [Issues]
root@K1Max-AC75 /root [#] git clone https://github.com/Guilouz/Crealit
cript.git /usr/data/helper-script
Cloning into '/usr/data/helper-script'...
remote: Enumerating objects: 276, done.
remote: Counting objects: 100% (276/276), done.
remote: Compressing objects: 100% (193/193), done.
Receiving objects:  45                                                % (126/276), 92.00 KiB                                                error: RPC failed; curl 18 transfer closed with outstanding read data remaining
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed

**Comment by firepower9966** [comment]
after multiple attempts it did complete.

**Reply by Guilouz** [reply]
It's an network issue.
